code/game/gamemodes/gang/gang_items.dm

```diff
@@ -312,36 +568,147 @@
 
-/datum/gang_item/equipment/implant_breaker
-	name = "Implant Breaker"
-	id = "implant_breaker"
-	cost = 10
+/datum/gang_item/equipment/gangbreaker
+	name = "Gangbreaker Implant"
+	id = "gangbreaker"
+	cost = 20
+	item_path = /obj/item/weapon/implanter/mindshield
+	spawn_msg = "<span class='notice'>Nanotrasen has provided you with a prototype mindshield implant that will both break a gang's control over a person and shield them from further conversion attempts.Gang bosses are immune.</b></u></span>"
+
+/datum/gang_item/equipment/recruiter
+	name = "Advanced Recruitment Implant"
+	id = "recruitment"
+	cost = 20
 	item_path = /obj/item/weapon/implanter/gang
-	spawn_msg = "<span class='notice'>The <b>implant breaker</b> is a single-use device that destroys all implants within the target before trying to recruit them to your gang. Also works on enemy gangsters.</span>"
+	spawn_msg = "<span class='notice'>The <b>Advanced Recruitment Implant</b> is a single-use device that is guaranteed to convert anyone into your gang, only rival gang bosses are immune.</b></u></span>"
 
-/datum/gang_item/equipment/implant_breaker/spawn_item(mob/living/carbon/user, datum/gang/gang, obj/item/device/gangtool/gangtool)
+/datum/gang_item/equipment/recruiter/spawn_item(mob/living/carbon/user, datum/gang/gang, obj/item/device/gangtool/gangtool)
 	if(item_path)
 		var/obj/item/O = new item_path(user.loc, gang) //we need to override this whole proc for this one argument
 		user.put_in_hands(O)
 	if(spawn_msg)
 		to_chat(user, spawn_msg)
-		
+
 /datum/gang_item/equipment/wetwork_boots
 	name = "Wetwork boots"
 	id = "wetwork"
 	cost = 20
 	item_path = /obj/item/clothing/shoes/combat/gang
-	
+
+//////// REVIVIFICATION SERUM //////////
+
+/datum/gang_item/equipment/reviver
+	name = "Outlawed Reviver Serum"
+	id = "reviver"
+	cost = 50
+	item_path = /obj/item/weapon/reviver
+
+/datum/gang_item/equipment/reviver/get_cost(mob/living/carbon/user, datum/gang/gang, obj/item/device/gangtool/gangtool)
+	cost = 10 + gang.gangsters.len * 2
+	return cost
+
+/obj/item/weapon/reviver
+	name = "outlawed revivification serum"
+	desc = "Banned due to side effects of extreme rage, reduced intelligence, and violence. For gangs, that's just a fringe benefit."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "implanter1"
+	item_state = "syringe_0"
+	throw_speed = 3
+	throw_range = 5
+	w_class = WEIGHT_CLASS_SMALL
+	origin_tech = "materials=2;biotech=5"
+	materials = list(MAT_METAL=600, MAT_GLASS=200)
+
+
+/obj/item/weapon/reviver/attack(mob/living/carbon/human/H, mob/user)
+	if(!ishuman(H))
+		return
+	user.visible_message("<span class='warning'>[user] begins inject [H] with [src].</span>", "<span class='warning'>You begin to inject [H] with [src]...</span>")
+	var/total_burn	= 0
+	var/total_brute	= 0
+	H.notify_ghost_cloning("You're being injected with a revivification serum!")
+	if(do_after(user, 30, target = H))
+		if(H.stat == DEAD)
```

Comment:
I poked around with this but couldn't quite get it working, do I have to make the checks into their own proc first?